src/Test/Controller/AdminControllerWebTestCase.php

```diff
@@ -60,24 +54,29 @@ protected function prepareAdminUrl(array $queryParameters): string
     }
 
     /**
-     * @param FormField|array<FormField>|FormField[][] $fields
+     * @param FormField|array<array-key, FormField>|array<array-key, array<array-key, FormField>> $fields
```

Comment:
The original annotation was not actually correct. The value of $fields can be N-deep array.